### ISSUE TITLE
A "Save File As" window pops up when add a new file in work space.

### ISSUE THREAD

**Issue by linette-zyy** [microsoft/PTVS#6240]
**Environment**
![image](https://user-images.githubusercontent.com/59192491/92367537-1c5c8500-f129-11ea-9355-10790c18950f.png)

**Steps to Reproduce**
1. Open a folder
2. Add new file in the folder
3. Check the whole screen

**Expected behavior**
A new file was added in the folder.

**Actual behavior **
A "Save File As" window pops up.
![image](https://user-images.githubusercontent.com/59192491/92368450-55e1c000-f12a-11ea-9e3a-b779416c7aba.png)

Note: This issue is not always occur. When restart VS, this issue is not repro.

**Comment by huguesv** [comment]
I'm unable to reproduce this. It is likely to be a bug in VS itself, not the Python workload, though I've searched the bug database and didn't find anything on this.

**Comment by huguesv** [comment]
@linette-zyy have you seen this recently?

**Comment by linette-zyy** [comment]
Yes, I saw it once recently. It usually happened when I always tested in one workspace without restart.

**Comment by linette-zyy** [comment]
Today, I have repro this issue. I have attached video.
Environment info
![image](https://user-images.githubusercontent.com/59192491/99506950-bceae300-29bd-11eb-91b9-b2366c44656e.png)
[Repro issue.zip](https://github.com/microsoft/PTVS/files/5558841/Repro.issue.zip)

**Comment by linette-zyy** [comment]
This issue is also reproduce today.
![image](https://user-images.githubusercontent.com/59192491/101469687-06f13400-3980-11eb-8ee4-ca4d5f728890.png)
